### ISSUE TITLE
FeatureRequest: Import custom prices

### ISSUE THREAD

**Issue by Sardaukai** [GoldenGnu/jeveassets#254]
It would be great if there is a way to import / update custom prices.
Then producers could instantly compare the market price and their custom (i.e. price to build that item)

Further it could be used to filter at Market Orders, maybe with an extra column named custom price.

A simple csv import with typeid and price should be fine and easy to manage

Cheers Sar

PS: I would not priories this, maybe you have some time left ;)

**Comment by ericjeker** [comment]
Yes, this would be great. I also need to import build prices to be able to calculate profit easily. But while discussing with Golden I was wondering if we couldn't not use evecookbook to request the build price via their API and using existing BPC/BPO that we have in our inventory? They do so in EVE IPH.

Is your usecase only for build price? Or also for buy price?

**Comment by Sardaukai** [comment]
At the moment I use buy price (Options -> Settings -> Values -> Prices) as a kind of build price to calculate sell profit
If there would be a build price it would be great too.

**Comment by TsuroTsero** [comment]
Starting to look at this
